docs/content/users/providers/acquia.md

```diff
@@ -23,7 +21,7 @@ DDEV’s Acquia integration pulls database and files from an existing project in
     ddev config global --web-environment-add="ACQUIA_API_KEY=xxxxxxxx,ACQUIA_API_SECRET=xxxxx"
     ```
 
-5. In the project `.ddev/config.yaml` add the `ACQUIA_ENVIRONMENT_ID` environment variable:
+3. In the project `.ddev/config.yaml` add the `ACQUIA_ENVIRONMENT_ID` environment variable:
```

Comment:
If you swap these all to `1.` you'll never have to update the numbering again. Markdown sees `1. 1. 1.` and renders out a correct ordered list.

Reply:
Not true in our case as the markdown rules we have set up require the more readable 1, 2, 3. But agreed that it wouldn't be unreasonable to change the rules. Our goal has been to make all the markdown readable, and the 1, 1, 1 technique is definitely not as readable (but lots easier to maintain).

Thanks for reviewing!